main.go

```diff
@@ -341,10 +341,11 @@ func main() { // nolint:gocyclo
 		os.Exit(1)
 	}
 	if err = (&spacebindingcleanup.Reconciler{
-		Client:         mgr.GetClient(),
-		Scheme:         mgr.GetScheme(),
-		Namespace:      namespace,
-		MemberClusters: clusterScopedMemberClusters,
+		Client:             mgr.GetClient(),
+		Scheme:             mgr.GetScheme(),
+		Namespace:          namespace,
+		MemberClusters:     clusterScopedMemberClusters,
+		PublicViewerConfig: crtConfig.PublicViewer(),
```

Comment:
I'm wondering if we should pass this here or read it at "Reconcile" time? 

In this way any change to this config would need a restart of the operator. I don't think there would be many changes but just something to consider.

Reply:
definitely at the reconcile time - we shouldn't need to restart the container when we change the configuration.
There is also a logic that wraps the ToolchainConfig, does some caching and provides the default values if nothing is set.

Reply:
@MatousJobanek, just to be sure, like what's done [here](https://github.com/codeready-toolchain/host-operator/blob/de95481affea496ceb043113a347f64a61abe42c/controllers/usersignup/usersignup_controller.go#L123).

out of curiosity - why not doing this with [k8s volume mounting](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-files-from-a-pod) and just fetch the data from filesystem?

Reply:
> like what's done [here](https://github.com/codeready-toolchain/host-operator/blob/de95481affea496ceb043113a347f64a61abe42c/controllers/usersignup/usersignup_controller.go#L123)?

yes that's one example of how to fetch the config/default values